### ISSUE TITLE
Missing reference in docs

### ISSUE THREAD

**Issue by incompletude** [cloudscribe/NoDb#4]
I think NoDb is the right choice for me, but I'm having trouble putting it to work.

The following reference is missing:

https://github.com/joeaudette/cloudscribe.SimpleContent/blob/master/src/cloudscribe.SimpleContent.Storage.NoDb/NoDbPageRepository.cs

Is there a step by step tutorial somewhere?

Thx

**Comment by joeaudette** [comment]
Hi,
Thanks for letting me know. The SimpleContent project has been refactored so queires and commands are in separate classes not combined together in a repository class. I've updated the links in the readme.

There isn't a step by step tutorial, but I've used NoDb extensively in my client projects with [cloudscribe Core](https://github.com/cloudscribe/cloudscribe) and [cloudscribe Simple Content](https://github.com/cloudscribe/cloudscribe.SimpleContent), and those repositories have good code examples that are used in the real world.

**Comment by incompletude** [comment]
I'm trying to do that, but is way above my level. A step by step tutorial would be nice. Thx anyways. Nice work.

**Comment by joeaudette** [comment]
@incompletude I'm more than happy to try to help you. Are you a student, new to c# or, at least if you can help me know where you are stuck in your understanding, I am happy to try to explain and maybe a tutorial would evolve out of that. I'm only guessing maybe it is the generic syntax like IBasicQueries of T that is hard to understand, I know C# generics was initially hard to understand for me but then became a eureka moment once I understood it.

In the generic syntax T is just your model class so it can be whatever your model needs to be, and your model can be persisted to file system storage. 

If you make a repository on github with a project trying to use NoDb then let me know the project url and what you are stuck on and I can see your code and make suggestions to help you along.

**Comment by incompletude** [comment]
Hey @joeaudette, thank you very much for your time and your assistance. 

I'm working on a new version of my portfolio that contains a blog and a few other areas. I first tried to build a static generator using gulp and webpack, but it got overcomplicated very fast. I'm now migrating to .NET Core (that I know well enough). 

My idea is to put the data I need inside JSON files, that I can easily consume throughout the application. Looking for some implementation of this idea, I arrived at your solution, and it seems exactly what I need.

Unfortunately, I was looking for something more straightforward, I have very limited time to work on this project, and now I realize NoDb requires me to dive in and understand the code architecture before being able to be productive at my original goal. It seems to me that even by having a basic database with some CRUD would be easier to do.

Again, I thank you very much for your time, your solution seems very elegant.
